src/main/java/net/openhft/chronicle/bytes/AbstractBytes.java

```diff
@@ -406,7 +406,10 @@ public byte readByte()
             return bytesStore.readByte(offset);
 
         } catch (BufferUnderflowException e) {
-            return 0;
+            if (lenient) {
+                return 0;
```

Comment:
This would be superslow as an exception is created on each read...

Reply:
Sure, but the parseDouble() no longer uses this method, it is the one for end-users of Bytes and in line with other methods available for them.